### ISSUE TITLE
RuntimeException finishRun failed

### ISSUE THREAD

**Issue by spokenbird** [dequelabs/axe-core-maven-html#190]
Getting a runtime exception when attempting to upgrade to version 4.3.0.

Our project is here: https://github.com/codeforamerica/shiba

The issue can be recreated by cloning the project and running `./gradlew clean accessibilityTest` from the root directory.

Also we noticed that the help link in the error message appears to be dead. (https://github.com/dequelabs/axe-core-maven-html/error-handling.md)

```
AccessibilityJourneyTest > laterDocsFlow() FAILED
    java.lang.RuntimeException: axe.finishRun failed. Please check out https://github.com/dequelabs/axe-core-maven-html/error-handling.md`
        at com.deque.html.axecore.selenium.AxeBuilder.analyzePost43x(AxeBuilder.java:616)
        at com.deque.html.axecore.selenium.AxeBuilder.analyzeRawContext(AxeBuilder.java:526)
        at com.deque.html.axecore.selenium.AxeBuilder.analyze(AxeBuilder.java:484)
        at org.codeforamerica.shiba.testutilities.AccessibilityTestPage.testAccessibility(AccessibilityTestPage.java:45)
        at org.codeforamerica.shiba.testutilities.AccessibilityTestPage.clickButton(AccessibilityTestPage.java:27)
        at org.codeforamerica.shiba.testutilities.Page.clickContinue(Page.java:69)
        at org.codeforamerica.shiba.journeys.AccessibilityJourneyTest.laterDocsFlow(AccessibilityJourneyTest.java:105)

        Caused by:
        org.openqa.selenium.JavascriptException: javascript error: Maximum call stack size exceeded
          (Session info: headless chrome=93.0.4577.63)
        Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
        System info: host: 'fv-az204-236', ip: '10.1.0.165', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1041-azure', java.version: '16.0.2'
        Driver info: org.openqa.selenium.chrome.ChromeDriver
        Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 93.0.4577.63, chrome: {chromedriverVersion: 93.0.4577.63 (ff5c0da2ec0ad..., userDataDir: /tmp/.com.google.Chrome.yaVN22}, goog:chromeOptions: {debuggerAddress: localhost:33737}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
        Session ID: 97e89563e2456f14eacaf4ab3ec2caec
            at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
            at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
            at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
            at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
            at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
            at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
            at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
            at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
            at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:489)
            at com.deque.html.axecore.extensions.WebDriverInjectorExtensions.executeScript(WebDriverInjectorExtensions.java:50)
            at com.deque.html.axecore.selenium.AxeBuilder.analyzePost43x(AxeBuilder.java:614)
            ... 6 more

AccessibilityJourneyTest > userCanCompleteTheNonExpeditedHouseholdFlow() FAILED
    java.lang.RuntimeException: axe.finishRun failed. Please check out https://github.com/dequelabs/axe-core-maven-html/error-handling.md`
        at com.deque.html.axecore.selenium.AxeBuilder.analyzePost43x(AxeBuilder.java:616)
        at com.deque.html.axecore.selenium.AxeBuilder.analyzeRawContext(AxeBuilder.java:526)
        at com.deque.html.axecore.selenium.AxeBuilder.analyze(AxeBuilder.java:484)
        at org.codeforamerica.shiba.testutilities.AccessibilityTestPage.testAccessibility(AccessibilityTestPage.java:45)
        at org.codeforamerica.shiba.testutilities.AccessibilityTestPage.clickButton(AccessibilityTestPage.java:27)
        at org.codeforamerica.shiba.testutilities.Page.clickContinue(Page.java:69)
        at org.codeforamerica.shiba.journeys.AccessibilityJourneyTest.userCanCompleteTheNonExpeditedHouseholdFlow(AccessibilityJourneyTest.java:169)

        Caused by:
        org.openqa.selenium.JavascriptException: javascript error: Maximum call stack size exceeded
          (Session info: headless chrome=93.0.4577.63)
        Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
        System info: host: 'fv-az204-236', ip: '10.1.0.165', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1041-azure', java.version: '16.0.2'
        Driver info: org.openqa.selenium.chrome.ChromeDriver
        Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 93.0.4577.63, chrome: {chromedriverVersion: 93.0.4577.63 (ff5c0da2ec0ad..., userDataDir: /tmp/.com.google.Chrome.yaVN22}, goog:chromeOptions: {debuggerAddress: localhost:33737}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
        Session ID: 97e89563e2456f14eacaf4ab3ec2caec
            at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
            at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
            at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
            at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
            at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
            at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
            at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
            at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
            at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
            at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:489)
            at com.deque.html.axecore.extensions.WebDriverInjectorExtensions.executeScript(WebDriverInjectorExtensions.java:50)
            at com.deque.html.axecore.selenium.AxeBuilder.analyzePost43x(AxeBuilder.java:614)
            ... 6 more
21:33:03.838 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
21:33:03.841 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':accessibilityTest'.

> Task :accessibilityTest FAILED
> There were failing tests. See the report at: file:///home/runner/work/shiba/shiba/build/reports/tests/accessibilityTest/index.html
7 actionable tasks: 7 executed

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 8s
```

**Comment by AdnoC** [comment]
Should be resolved by https://github.com/dequelabs/axe-core-maven-html/pull/191

**Comment by Zidious** [comment]
Closing due to inactivity, please re-open is issue is apparent in version: 4.4.1
